src/kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -184,6 +184,11 @@ impl CodegenBackend for GotocCodegenBackend {
 }
 
 fn check_options(session: &Session, need_metadata_module: bool) {
+    // The requirement below is needed to build a valid CBMC machine model
```

Comment:
Nit: can we point/link to/reference between these two things.

e.g. mention `machine_model_from_session` here.

You already back-reference to here from the other spot, just completing the circle :)

Reply:
Maybe add a link to an issue too.